src/test/kotlin/fi/metatavu/timebank/api/test/functional/impl/TimeEntriesTestBuilderResource.kt

```diff
@@ -20,7 +20,7 @@ class TimeEntriesTestBuilderResource(
 ): ApiTestBuilderResource<TimeEntry, ApiClient?>(testBuilder, apiClient) {
 
     override fun clean(t: TimeEntry) {
-        api.deleteTimeEntry(t.id!!)
```

Comment:
Usually resources are deleted by the id, could you explain logic behind this change?